src/helm/benchmark/scenarios/audio_language/audio_mnist_scenario.py

```diff
@@ -37,28 +42,21 @@ class AudioMNISTScenario(Scenario):
     }
     """  # noqa: E501
 
-    NUM_SPEAKERS = 60
-    NUM_TRIALS = 50
-    WAV_URL_TEMPLATE = r"https://github.com/soerenab/AudioMNIST/raw/544b0f4bc65227e54332e665d5e02c24be6732c2/data/{speaker_id}/{digit}_{speaker_id}_{trial_index}.wav"  # noqa: E501
-
     name = "audio_mnist"
     description = "Classify an audio sample of a spoken digit ([Becker et al, 2023](https://arxiv.org/abs/1807.03418))."
     tags = ["audio", "classification"]
 
     def get_instances(self, output_path: str) -> List[Instance]:
         instances: List[Instance] = []
-        for digit in range(10):
-            for speaker_index in range(AudioMNISTScenario.NUM_SPEAKERS):
-                speaker_id = str(speaker_index).zfill(2)
-                for trial_index in range(AudioMNISTScenario.NUM_TRIALS):
-                    wav_url = AudioMNISTScenario.WAV_URL_TEMPLATE.format(
-                        digit=digit, speaker_id=speaker_id, trial_index=trial_index
-                    )
-                    input = Input(
-                        multimedia_content=MultimediaObject([MediaObject(content_type="audio/wav", location=wav_url)])
-                    )
-                    references = [Reference(Output(text=str(digit)), tags=[CORRECT_TAG])]
-                    # Don't need train split because we're using zero-shot
-                    instance = Instance(input=input, references=references, split=TEST_SPLIT)
-                    instances.append(instance)
+        wav_save_dir: str = os.path.join(output_path, "wav_files")
+        ensure_directory_exists(wav_save_dir)
+        for row in tqdm(load_dataset("flexthink/audiomnist", cache_dir=output_path, split=TEST_SPLIT)):
```

Comment:
Why use a Hugging Face mirror instead of the original dataset? The original dataset has links to raw wav files and it's much more straightforward to just download those files rather than convert them from `datasets`.

Reply:
I tried to download Audio-MNIST from the original GitHub repo, it takes around 2-3 hrs to download the full dataset using `ensure_file_downloaded` from GitHub. But I can download the full dataset from HF mirror in less than 1 min, and choose a split (test split in audio-mnist, 750 examples) to write to the local disk, which takes little time.

Reply:
Sounds good, I will double check this on my end. Let's stick with HF then.